conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -270,13 +270,13 @@ func (algodImp *algodImporter) getDelta(rnd uint64) (sdk.LedgerStateDelta, error
 	params := struct {
 		Format string `url:"format,omitempty"`
 	}{Format: "msgp"}
-	bytes, err := (*common.Client)(algodImp.aclient).GetRaw(algodImp.ctx, fmt.Sprintf("/v2/deltas/%d", rnd), params, nil)
+	// Note: this uses lenient decoding. GetRaw and msgpack.Decode would allow strict decoding.
+	err := (*common.Client)(algodImp.aclient).GetRawMsgpack(algodImp.ctx, &delta, fmt.Sprintf("/v2/deltas/%d", rnd), params, nil)
 	if err != nil {
-		return delta, err
+		return sdk.LedgerStateDelta{}, err
 	}
 
-	err = msgpack.Decode(bytes, &delta)
-	return delta, err
+	return delta, nil
```

Comment:
This is the source of the error I was running into. The master branch of algod has a new field in `ledgercore.StateDelta` and the SDK wasn't able to parse it with the strict decoder.